gyft.py

```diff
@@ -72,6 +72,20 @@
 rows = rows_head.findAll('tr')
 times = []
 
+
+# Delete the rows that doesn't have tableheader, basically without a weekday
+# this is done because the erp does shit see issues and solution
+del_rows = []
+for i in range(1, len(rows)):
+    HeaderRows = rows[i].findAll("td", {"class": "tableheader"})
+    # print(HeaderRows)
+    if len(HeaderRows) is 0:
+        del_rows.append(i)
+
+for index_del in del_rows:
+    del rows[index_del]
```

Comment:
This might fail if there is more than one element in `del_rows`.
It should be something like:
```
for index_del in sorted(del_rows, reverse=True):
    del rows[index_del]
```